src/main/java/gregtech/common/metatileentities/electric/MetaTileEntityBatteryBuffer.java

```diff
@@ -81,4 +86,14 @@ protected ModularUI createUI(EntityPlayer entityPlayer) {
             .bindPlayerInventory(entityPlayer.inventory)
             .build(getHolder(), entityPlayer);
     }
+
+    @Override
+    public void addInformation(ItemStack stack, @Nullable World player, List<String> tooltip, boolean advanced) {
+        String tierName = GTValues.VN[getTier()];
+
```

Comment:
might be good to print max output amperage here too

Reply:
No problem done